qdrant-landing/themes/qdrant-2024/layouts/partials/qdrant-blog-posts.html

```diff
@@ -0,0 +1,30 @@
+{{ with (.Site.GetPage "/qdrant-blog/qdrant-blog-posts") }}
+  <section class="qdrant-blog-posts">
+    <div class="container">
+      <div class="qdrant-blog-posts__articles post">
```

Comment:
@nastyapash I suppose that you preferred custom styles over using bootstrap `.row > .col` classes to make gutters between posts wider as in the design 🙂 
But here is a way to do the same with Bootstrap (`.row .gx-5`) https://getbootstrap.com/docs/5.1/layout/gutters/

Could we please use this? 🙏

Reply:
done!